wordpress-mobile/WordPress-iOS
# Adds some missing translations for the iOS 14 All Time widget

Fixes #NA

This PR adds translations for "Best views ever" and "Posts" in the medium size All Time widget.

To test:
- build, run and install a medium size All Time widget
- set any language on your device and make sure that the translations of "Best views ever" and "Posts" look correct. They can me matched against the `Localizable.strings` files that can be found in the `WordPress` target.

Notes:
- some languages did not have translations for one or both the two strings, in this case it will default to English
- the Italian translation of "Best views ever" in the widget will be different than the one in the main app (only exception): the translation was replaced with a shorter one with the same meaning.

PR submission checklist:


cc @jkmassel , this targets release 16.9.
